src/adapter/etl-adapter-excel/tests/Flow/ETL/Adapter/Excel/Tests/Integration/ExcelExtractorTest.php

```diff
@@ -175,4 +175,26 @@ public function test_extract_with_wrongly_selected_reader() : void
 
         iterator_to_array($extractor->extract(flow_context(config())));
     }
+
+    public function test_loading_data_from_all_partitions() : void
+    {
+        self::assertSame(
+            [
+                ['group' => 1, 'id' => 1, 'value' => 'a'],
+                ['group' => 1, 'id' => 2, 'value' => 'b'],
+                ['group' => 1, 'id' => 3, 'value' => 'c'],
+                ['group' => 1, 'id' => 4, 'value' => 'd'],
+                ['group' => 2, 'id' => 5, 'value' => 'e'],
+                ['group' => 2, 'id' => 6, 'value' => 'f'],
+                ['group' => 2, 'id' => 7, 'value' => 'g'],
+                ['group' => 2, 'id' => 8, 'value' => 'h'],
+            ],
+            df()
+                ->read(from_excel(__DIR__ . '/../Fixtures/partitioned/group=*/*.xlsx'))
+                ->withEntry('id', ref('id')->cast('int'))
+                ->sortBy(ref('id'))
+                ->fetch()
```

Comment:
you should firs just assign result of `->fetch()` to `$rows` variable and check if partition is available on it - that's what I meant in the comment about about missing test